src/connection.ts

```diff
@@ -41,6 +41,7 @@ import { createNTLMRequest } from './ntlm';
 import { ColumnEncryptionAzureKeyVaultProvider } from './always-encrypted/keystore-provider-azure-key-vault';
 
 import { AbortController, AbortSignal } from 'node-abort-controller';
+import AbortError from './errors/abort-error';
```

Comment:
You might be able to remove this import if you remove the cases where you construct `AbortError` instances.